CONTRIBUTING.md

```diff
@@ -0,0 +1,139 @@
+# Contributing to Extension Activity Monitor
+
+Firstly, thank you for your interest in contributing to Extension Activity Monitor. Your contribution will make this project more awesome. 🚀
+
+## Table of Contents
+
+- [Project Architecture](#project-architecture)
+  - [Data Flow](#data-flow)
+  - [Project Directory Structure](#project-directory-structure)
+- [Pick an issue](#pick-an-issue)
+- [Installation](#installation)
+- [Run Test](#run-test)
+- [Check for Linter](#check-for-linter)
+- [Check for Prettier](#check-for-prettier)
+- [Submit a Bug or Issue or Feature Request](#submit-a-bug-or-issue-or-feature-request)
+- [Create a Pull Request](#create-a-pull-request)
+  - [Write Commit Message](#write-commit-message)
+
+## Project Architecture
+
+This extension is primarily consist of a Popup (browserAction Popup) and Activity Log page (Extension page). The popup helps to "start monitor" or "stop monitor" all extensions and access the activity log page. On the other hand, the Activity Log page helps to view the logs detail, filtering logs, save the logs and load logs from a file.
+
+The Activity Log page is using the MVC architecture and [web components](https://github.com/mozilla/extension-activity-monitor/tree/master/src/lib/web-component) for table view and filtering options. ES6 classes/modules are being used to encapsulate the different functionalities of the extension. Unit test is done with [JEST Framework](https://jestjs.io/) in this project.
```

Comment:
"ES6 classes/modules are being used ..." can be removed - it is obvious after one look at the code.

The reference to the Jest framework can be removed to a separate section (about testing). You should have one section on testing that explains which test framework is used and how to run them.